index.html

```diff
@@ -165,10 +165,41 @@ <h3 class="proj-title">Tonic</h3>
          <li>css</li>
          <li>javascript</li>
         </ul>
-        <button type="button" class="poppins-font button">See Project</button>
+        <button type="button" class="poppins-font button see-project-button" id="project4" >See Project</button>
+      </div>
+     </section>
+  </div>
+
+  <!-- Project detail popup -->
+  <div class="project-popupbg" id="popup-1">
+    <!-- removing  project class from section element-->
+    <section class="container-bg-color popup-detailCard">
+      <div class="prj-popup-header">
+        <i class="fas fa-times x-icon" id="x-icon-popup-id"></i>
+        <h3 class="proj-title" id="proj-title-id"></h3>
+      </div>
+      <ul class="proj-about no-bullets" id="proj-about-id">
+        <li class="proj-name" id="proj-name-id"></li>
+        <li class="proj-type" id="proj-type-id"></li>
+        <li class="proj-year" id="proj-year-id"></li>
+      </ul>
+      <img src="#" alt="protfolio snapshot" class="img popup-detailImg" id="img-id">
+      <div class="popup-txtarea">
+        <p class="proj-primarytxt" id="proj-primarytxt-id"></p>
+        <div id="popuplist">
+          <ul class="no-bullets proj-list" id="tech-list-id">
+          </ul>
+          <div class="style-popupbtn">
+            <button type="button" class="poppins-font button" id="sourcebtn-id">
+              See Source <img src="./images/purple-github.svg" alt="github icon"> </button>
+            <button type="button" class="poppins-font button" id="livebtn-id">
+              See Live <img src="./images/seelive-icon.svg" alt="seelive icon"> </button>
+          </div>
+        </div>
       </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** kindly make sure your Popup in mobile matches the Figma design as required. The project's image has to be displayed before the text. Here is how your design looks:

| Figma Design | Your Design |
| ----------------------- | --------------------- |
| ![Screenshot from 2022-07-21 18-02-54](https://user-images.githubusercontent.com/83042742/180624176-948b9415-9045-4335-b3d3-987ef0c5defc.png) | ![Screenshot from 2022-07-23 23-47-24](https://user-images.githubusercontent.com/83042742/180624042-521b2997-32b3-4dd1-916d-3cf79bf3438a.png) |

### Notice: Make sure you implement this change before you merge your pull request. This should be required by the previous reviewer.

Reply:
Thanks for the review @tresorsawasawa i changed that